app/components/BottomNavigation.tsx

```diff
@@ -42,7 +49,7 @@ const BottomNavigation = () => {
           </Link>
 
           <button
-            onClick={handleChatClick}
+            onClick={() => setIsChatVisible(true)}
```

Comment:
This make the UI quite ugly when there is a "Loading chat" in the middle of the view when users clicking this button. Let's do these to make it better:
1. Load the Chat component when hovering the button
2. No need to display "Loading chat" UI

That way once users click the chat, JS code should be loaded faster and hence create a more seamless experience